pages/profile.tsx

```diff
@@ -2,54 +2,109 @@ import { Chess } from "chess.js";
 import type { NextPage } from "next";
 import ChessBoard from "../components/chessboard/ChessBoard";
 import Link from "next/link";
+import { useContext } from "react";
+import { UserInfoContext } from "../context/UserInfo";
+import CircularLoader from "../components/CircularLoader";
+import { useForm, SubmitHandler } from "react-hook-form";
+
+type Inputs = {
+  name: string;
+  boardLightColor?: string;
+  boardDarkColor?: string;
+};
 
 const Profile: NextPage = () => {
+  const userInfo = useContext(UserInfoContext);
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<Inputs>();
+  //TODO implement saving
+  const onSubmit: SubmitHandler<Inputs> = (data) => console.log(data);
+  console.log(userInfo?.user);
+
+  if (!userInfo?.user) return <CircularLoader />;
+  let lightColor = userInfo.user.boardLightColor ?? "white";
+  let darkColor = userInfo.user.boardDarkColor ?? "#FCA311";
   return (
     <div className="container text-center p-4">
-      <div className="row">
+      <div className="row mb-5">
         <div className="col">
-          <h1 className="mb-4">Your Profile</h1>
-          <img src="/profile-avatar.png" className="img-fluid mb-5" />
-          <h5 className="text-start fw-semibold">User tier:</h5>
-          <h3 className="fw-semibold mt-4">Basic Membership</h3>
-          <Link href="/upgrade" passHref>
+          <form onSubmit={handleSubmit(onSubmit)}>
+            <h1 className="mb-4">Your Profile</h1>
+            <img src="/profile-avatar.png" className="img-fluid mb-5" />
```

Comment:
Maybe user the user image directly here & style it a bit? 

```js
<img src={userInfo.user.image} className="img-fluid mb-5" />
```

should get the job done ^^

Reply:
Oh yeah definitely will add that next thanks!